code/modules/food_and_drinks/plate.dm

```diff
@@ -60,6 +57,15 @@
 	SIGNAL_HANDLER
 	ItemRemovedFromPlate(moved_item)
 
+/obj/item/plate/proc/if_can_move(obj/item/I, mob/user)
```

Comment:
I'd prefer this proc be named something like `can_hold_item()` or `can_accept_item()`